powa/static/js/components/Grid.vue

```diff
@@ -230,16 +220,13 @@ function getAlign(type) {
       return "right";
   }
 }
+</script>
 
-function onRowClicked(row) {
-  if (row.url) {
-    window.location.href = [row.url, serialize(store.from, store.to)].join("?");
-  }
+<style lang="scss">
+td a {
+  text-decoration: none;
 }
-
-function rowClass(row) {
-  return row.url ? "clickable" : "";
```

Comment:
You may need to remove the corresponding rule in `main.scss` since it's not used anywhere else.